src/wp-includes/rest-api/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -1681,7 +1681,7 @@ public function get_collection_params() {
 			'default'           => 'approve',
 			'description'       => __( 'Limit result set to comments assigned a specific status. Requires authorization.' ),
 			'sanitize_callback' => 'sanitize_key',
-			'type'              => 'string',
+			'type'              => array( 'string', 'array' ),
```

Comment:
We can just say `array`. The API will accept a string, and convert it to an array of items.

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/9870/commits/f121b3e3d930f221e5324168a75f28c0e23acaef